uber/models.py

```diff
@@ -552,6 +529,7 @@ def update_badge(self, attendee, old_badge_type, old_badge_num):
             :param old_badge_num: The old badge number.
             :return:
             """
+            from uber.badge_funcs import needs_badge_num
```

Comment:
we could put it back in session and just make it a static member also.  there's no reason this shouldn't be ok though except for this weird import stuff.